src/vaadin-confirm-dialog.html

```diff
@@ -27,6 +27,7 @@
       id="dialog"
       opened="{{opened}}"
       aria-label="[[_getAriaLabel(header)]]"
+      theme="[[theme]]"
```

Comment:
`theme` property (on `vaadin-dialog` and other Vaadin components) is a read only property which can only be set via attribute so this should use an attribute binding instead. Should be changed to:
```html
theme$="[[theme]]"
```